draft-ietf-rtcweb-jsep.xml

```diff
@@ -697,8 +697,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           control how aggressively it will try to BUNDLE media streams together.
           The set of available policies is as follows:
           <list style="hanging">
-              <t hangText="balanced:">The application will BUNDLE all media streams of
-              the same type together. That is, if there are multiple audio and
+              <t hangText="balanced:">The application will bundle all media streams into
+              a single bundle group. All but the first media stream of each
+              type will be marked as bundle-only. That is, if there are multiple audio and
```

Comment:
Oh, I see what you are doing here.  That's a little confusing.  You are offering to unbundle different types, but allowing them to be ALL bundled.